src/please.go

```diff
@@ -676,6 +679,35 @@ func please(tid int, state *core.BuildState, parsePackageOnly bool, include, exc
 	}
 }
 
+// set the watch
+func setWatchedTarget(state *core.BuildState, labels []core.BuildLabel) string {
+	if opts.Watch.Run {
+		if len(labels) > 1 {
+			opts.Run.Parallel.PositionalArgs.Targets = opts.Watch.Args.Targets
+		} else {
+			opts.Run.Args.Target = opts.Watch.Args.Targets[0]
+		}
+		return "run"
```

Comment:
I don't think this is always right - if there is more than one label it needs to return "parallel"